src/com/maddyhome/idea/vim/group/MotionGroup.java

```diff
@@ -603,7 +603,7 @@ public boolean scrollColumnToLastScreenColumn(@NotNull Editor editor) {
 
   public static void scrollCaretIntoView(@NotNull Editor editor) {
     final EnumSet<CommandFlags> flags = CommandState.getInstance(editor).getExecutingCommandFlags();
-    final boolean scrollJump = flags.contains(CommandFlags.FLAG_IGNORE_SCROLL_JUMP);
+    final boolean scrollJump = !flags.contains(CommandFlags.FLAG_IGNORE_SCROLL_JUMP);
```

Comment:
This is correct, and was one of the bugs that started me down the rabbit hole :)

It's a flag saying we do scroll jump, because the "ignore scroll jump" flag is NOT set.